README.md

```diff
@@ -20,7 +20,7 @@
 
 # [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#rudderstack-javascript-sdk)RudderStack JavaScript SDK
 
-The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `rudder-analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
+The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
```

Comment:
The first sentence here doesn't make any sense to me. We should rather just say this,

```
The RudderStack JavaScript SDK helps you to track and send user events...
```

Reply:
As there is no such thing as `analytics.js` or `rudder-analytics.js` library. It's just a source code file in this repository and contributes to the final SDK.

Reply:
got it - thanks @saikumarrs - I've made the necessary update.